.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ on:
 
 jobs:
   test:
-    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v3
+    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v4.1.0
```

Comment:
Please revert this.

```suggestion
    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v3
```

Reply:
Thanks Aras, that's now been reverted. This allowed the previous CI run to pass without running on Mac v14 and v16, I'm expecting it will fail again as CI will run again on Mac for these versions.

Reply:
It's wild out of anyone to review this to be you @Uzlopak, this relates to a change you made in the `@octokit/webhooks` library, in the issue I linked above[0] that I've also created an issue and PR there for.

[0] https://github.com/octokit/webhooks.js/issues/1009